streamingpro-mlsql/src/main/java/streaming/rest/RestController.scala

```diff
@@ -423,6 +424,11 @@ class RestController extends ApplicationController with WowLog {
 
   //end -------------------------------------------
 
+  @At(path = Array("/listener/list"), types = Array(GET, POST))
```

Comment:
We should not add rest api in RestController. We recommend using SQL like this to get what you need:

```
load __mlsql__.`/listeners` as output;
```

Reply:
ok。